swift-sdk/Internal/AuthManager.swift

```diff
@@ -7,18 +7,20 @@ import Foundation
 
 @objc public protocol IterableInternalAuthManagerProtocol {
     func getAuthToken() -> String?
-    func requestNewAuthToken()
+    func requestNewAuthToken(_ hasFailedPriorAuth: Bool)
```

Comment:
I think the first parameter named should be included. It makes the intention of the method clearer. Otherwise, we don't know what we are setting to `true` or `false`.